For Issue #264

imran3004 opened the pull request:
@Jeff-Thompson12 I have implemented the changes for fixing issue #264 to this new branch originates in `dev`. This should fix the whitespace issues.

Jeff-Thompson12 reviewed (changes requested):
@imran3004 LGTM! Only thing missing is running `usethis::use_version("dev")` and updating the NEWS.md

imran3004 commented:
@Jeff-Thompson12 I have updated version with `usethis::use_version("dev")` and also updated the NEWS.md with a description of the change. Commit ID - b3324f2

Jeff-Thompson12 commented on DESCRIPTION:
```diff
@@ -1,6 +1,6 @@
 Package: riskassessment
 Title: A web app designed to interface with the `riskmetric` package
-Version: 0.0.1.9005
+Version: 0.0.1.9007
```
Should just be incremented to `9006`

Jeff-Thompson12 commented on NEWS.md:
```diff
@@ -17,3 +17,8 @@
 
 * Added a `NEWS.md` file to track changes to the package.
 * Used `{golem}` to complete "packagization" of `{riskassessment}`
+
+
+# riskassessment 0.0.1.9007
+
```
These lines are not needed. Comment should be added after line 9

Jeff-Thompson12 reviewed (changes requested):
Please see comments below.

Jeff-Thompson12 reviewed (approved):
LGTM!
